.github/workflows/ci.yml

```diff
@@ -13,6 +13,12 @@ on:
       - ".github/workflows/composite-cmake/**"
   pull_request:
 
+env:
+  # Due to busy file system problems on the CI runners, we introduced a patch to
+  # sleep if the MPI count of an I/O operation is unexpectedly 0 and then retry
+  # the I/O operation once. The time is in milliseconds.
+  IO_SLEEP_TIME: 5
```

Comment:
Please prefix with SC_ for a consistent name space.

Reply:
Done